shenyu-spring-boot-starter/shenyu-spring-boot-starter-client/shenyu-spring-boot-starter-client-springcloud/src/main/java/org/apache/shenyu/springboot/starter/client/springcloud/ShenyuSpringCloudClientConfiguration.java

```diff
@@ -27,12 +27,15 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.env.Environment;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 
 /**
  * The type shenyu spring cloud client configuration.
  */
 @Configuration
 @ImportAutoConfiguration(ShenyuClientCommonBeanConfiguration.class)
+@ConditionalOnProperty(value = "shenyu.register.enabled",
```

Comment:
Hi， mybe modify this ：     @ConditionalOnProperty(name = "shenyu.register.enabled", havingValue = "true", matchIfMissing = true)

and you can complete the registration switch for other clients and update the documentation?

Reply:
Maybe it's better to put it in `shenyu-client-springcloud`? `shenyu-spring-boot-starter-client-springcloud` is only used as the wrapper of `shenyu-client-springcloud`.

Reply:
Why?

Reply:
> Maybe it's better to put it in `shenyu-client-springcloud`? `shenyu-spring-boot-starter-client-springcloud` is only used as the wrapper of `shenyu-client-springcloud`.

I want to know what you think

Reply:
mybe add spring-boot-starter is the right

Reply:
got it